redash/query_runner/cass.py

```diff
@@ -22,6 +25,26 @@ def default(self, o):
         return super(CassandraJSONEncoder, self).default(o)
 
 
+def generate_cert_file(ssl_cert, host):
+    cert_dir = f"./tmp/cassandra_certificate/{host}"
```

Comment:
I think it will be safer to use a random file name + clean up after use. Maybe use [`NamedTemporaryFile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile)?

Reply:
ah cool, will use that.